extension/src/experiments/quickPick.ts

```diff
@@ -21,7 +21,7 @@ export const pickExperiment = (
         label,
         value: { id, name: name || label }
       })),
-      { title }
+      { matchOnDescription: true, matchOnDetail: true, title }
```

Comment:
I set `matchOnDescription` and `matchOnDetail` to be true for experiment selection, but we could also set it by default or on other commands like applying filters.